serverless.yml

```diff
@@ -65,67 +65,60 @@ functions:
   getTechRecords:
     handler: src/handler.handler
     events:
-    - http:
-        path: vehicles/{searchIdentifier}/tech-records
-        method: get
-        request:
-          parameters:
-            paths:
-              searchIdentifier: true
-            querystrings:
-              status: false
-              metadata: false
-              filename: false
-        documentation:
-          summary: "Get Vehicle Technical Records list"
-          description: "Retrieves the Vehicle Technical Records for the input searchIdentifier, which can be the VRM, the last 6 digits of the VIN or the full VIN"
-          parameters:
-            - name: searchIdentifier
-              in: path
-              description: The attribute to search a vehicle technical record by, which can be either the VRM, the last 6 digits of the VIN or the full VIN"
-              required: true
-              schema:
-                type: string
-            - name: status?
-              in: path
-              description: The status code of a vehicle technical record"
-              required: true
-              schema:
-                type: string
-            - name: metadata?
-              in: path
-              description: Applicable only to ADR details. Query param for returning the list of values that are selectable on the front end
-              required: false
-              schema:
-                type: string
-            - name: filename?
-              in: path
-              description: Applicable only to ADR details. Query param for downloading a document from S3. If specified, GET endpoint will return only the Base64 encoded string
-              required: false
-              schema:
-                type: string
```

Comment:
I know this isn't used  in the proper Serverless way, but it's still correct, right?

Reply:
What exactly are you referring to? I removed the "filename" path parameter from the getTechRecords endpoint because it's not used anymore. The other difference you're seeing is just indentation

Reply:
Oh, my mistake.